src/script/splide.js

```
@@ -0,0 +1,17 @@
+import Splide from '@splidejs/splide';
+
+document.addEventListener('DOMContentLoaded', function () {
+    new Splide('.splide', {
+        perPage: 1,
+    })
+        .on('pagination:mounted', function (data) {
+            data.list.classList.add('testimonials__pagination');
+
+            data.items.forEach((item, index) => {
+                item.button.classList.add(
+                    'testimonials__pagination--' + (index + 1),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)